rpm/dtc.spec

```diff
@@ -10,6 +10,7 @@ License:    GPLv2
 
 BuildRequires: flex
 BuildRequires: bison
+Requires:      ldconfig
```

Comment:
This should be: 
`Requires(post): /sbin/ldconfig
Requires(postun): /sbin/ldconfig
`

Reply:
Hmm, glibc provides the cap 'ldconfig', but i guess the '/sbin/ldconfig' works as well -- https://build2.jollamobile.com/package/binary/mer:core/glibc?arch=armv8el&filename=glibc-2.25%2Bgit4-1.9.2.jolla.armv7hl.rpm&repository=latest_armv7hl